styles.css

```diff
@@ -0,0 +1,50 @@
+/* Add a black background color to the top navigation */
+.topnav {
+    background-color: #333;
```

Comment:
I kindly suggest using two spaces instead of just one tap click for the reason of readability.